### PR TITLE
CVE fixes for User Profile Api

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-9478

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -12,8 +12,8 @@ def component = "user-profile-api"
 
 // Variables to switch pipeline logic and wiring per type of build
 def definitionStoreDevelopPr = "PR-575" // This doesn't change frequently, but when it does, only change this value.
-def userProfileApiDevelopPr = "PR-224" // This doesn't change frequently, but when it does, only change this value.
-def prsToUseAaat             = "PR-296" // Set this value to a PR number, or add it as a comma-separated value, if it's to follow CI/CD.
```
Comment (tomxelliott) [inline Jenkinsfile_CNP:16]: Do you want to change this or do you want to make it comma separated like the below?
`def prsToUseAaat             = "PR-296, PR-306"`

Comment (MSancaktutar) [reply Jenkinsfile_CNP:16]: def prsToUseAaat = "PR-306" is better, since 296 was already merged.

Comment (tomxelliott) [reply Jenkinsfile_CNP:16]: I wasn't aware of the status of 296 at the time of commenting. That makes sense 👍

**Comment by MSancaktutar** [comment]
Closing this PR as the plan has changed to release this with 20.1.0.

**Pull request closed without merging**